examples/example_for_base_functionality.py

```diff
@@ -38,12 +38,12 @@
     dataset.save(save_dir)
 
     # Copy data from "source_data_raw" to a "sds_dataset" parent directory adhering to SDS framework.
-    dataset.add_data(source_path="./test_data/sample1/raw", subject="subject-xyz", sample="sample-1", data_type="primary", sds_parent_dir=save_dir)
+    dataset.add_simples(source_path="./test_data/sample1/raw", subject="subject-xyz", sample="sample-1", data_type="primary", sds_parent_dir=save_dir)
```

Comment:
typo
should be sample(s)

Reply:
fixed by [8568637](https://github.com/SPARC-FAIR-Codeathon/sparc-me/pull/92/commits/856863752c90fad72015559f05f71db91768eca1)